content/laptop-battery-thresholds.md

```diff
@@ -20,6 +22,15 @@ The default charging profile for System76 laptops with Open Firmware has a start
 
 To determine if your laptop has Open Firmware or proprietary firmware, see [this article](/articles/open-firmware-systems). (If a system has Open Firmware, then it must also have Open EC to work with charging thresholds.) See [Charging Thresholds](#configuring-charging-thresholds-open-firmware) for Open Firmware systems or [FlexiCharger](#configuring-flexicharger-proprietary-firmware) for proprietary firmware systems.
 
+## Charging Thresholds or FlexiCharge support
+
+| Model | Codename | Charging Thresholds | FleixCharger |
+|:-----:|:---------:|:---------:|:---------:|
+| Pangolin | pang12 | No | No |
+| Pangolin | pang13 | No | No |
+| Pangolin | pang14 | No | No |
+| Pangolin | pang15 | No | No |
```

Comment:
I feel like if we're going to have a table with both Flexi and Threshold columns, we should make it a more exhaustive model list. Otherwise, this is no better than simply updating the note that was there before.